### PR TITLE
test: add testMintFee_WhenRampingA

### PR DESCRIPTION
**Pull request by andersonaguiar**

### CODE REVIEW

**Review by xenide — COMMENTED** [review]
yes this time you have changed the A by shifting the time, good job.

However, if there's no swaps, the liquidity and therefore `D` did not grow. So we cannot tell if the implementation has succeeded. We need to perform several swaps (can put them in a loop) to grow the liquidity. (see point 2 of my prev PR comment). 

Another way to do it is, create another stable pair, so we have `StablePair1` and `StablePair2`. Both of them experience the same liquidity changes (growth in D). But one has a ramping A, the other has a static A. Then we do a `mint` / `burn` / `mintFee` again. Then we see if the platform fee collector received the same amount of LP tokens as platform fees.

File: test/__fixtures/BaseTest.sol [hunk 1]
```diff
@@ -27,6 +27,7 @@ abstract contract BaseTest is Test {
     constructor()
     {
         // set shared variables
+        _factory.set(keccak256("ConstantProductPair::platformFeeTo"), bytes32(uint256(uint160(_platformFeeTo))));
         _factory.set(keccak256("ConstantProductPair::swapFee"), bytes32(uint256(30)));
         _factory.set(keccak256("ConstantProductPair::platformFee"), bytes32(uint256(2500)));
         _factory.set(keccak256("ConstantProductPair::defaultRecoverer"), bytes32(uint256(uint160(_recoverer))));
```
Comment (OliverNChalk) [inline test/__fixtures/BaseTest.sol:33]: Thoughts on refactoring this to use library? Would be good to lock in/test the library approach

**Review by OliverNChalk — APPROVED** [review]

File: test/__fixtures/BaseTest.sol (hunk 1 above)
Comment (xenide) [reply test/__fixtures/BaseTest.sol:33]: actually you didn't have to review this PR cuz it was the other one that mattered. I was going to close this PR. 

By library you mean like managing the setting + casting of these variables?

Comment (OliverNChalk) [reply test/__fixtures/BaseTest.sol:33]: yes

Comment (xenide) [reply test/__fixtures/BaseTest.sol:33]: will fix in a separate PR as this PR is old and no longer relevant

**Comment by xenide** [comment]
closing as this has been addressed in #86

**Pull request closed without merging**